File: src/main/java/com/sm/carwashmonitor/dto/WashCycleDTO.java
package com.sm.carwashmonitor.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class WashCycleDTO {
    private Long washCycleId;
    private Long stationId;
    private Long unitId;
    private String washCycleDate;
    private Integer coinAmount;
    private Float waterConsumption;
    private Float detergentConsumption;
    private Float waxConsumption;
}

File: carwash-backend/Dtos/WashCycleDto.cs
namespace CarwashMonitor.Dtos;

public class WashCycleDto
{
    public Guid Id { get; set; }
    public float? WaterConsumption { get; set; }
    public float? DetergentConsumption { get; set; }
    public float? WaxConsumption { get; set; }
    public int? CoinAmount { get; set; }
    public DateTime? DateCreated { get; set; }
}